models/utils/detect_face.py

```diff
@@ -306,6 +308,45 @@ def imresample(img, sz):
     return im_data
 
 
+def batch_resample_by_size(imgs, target_size, device):
+    """
+    Batch resampling function grouping by size while preserving order.
+
+    Args:
+    imgs (list of torch.Tensor): List of image tensors
```

Comment:
This line and those below need an additional indent

Reply:
done